vue-client/src/utils/data.js

```diff
@@ -205,3 +205,22 @@ export function xmlToJson(xml) {
   }
   return obj;
 }
+
+export function translateAliasedUri(uri) {
+  let translatedUri = uri;
+  
+  each(JSON.parse(process.env.VUE_APP_SITE_ALIAS || '{}'), (from, to) => {
+    if (uri.startsWith(from)) {
+      translatedUri = uri.replace(from, to);
+      return false;
+    }
+    return true;
+  });
+
+  // TODO: why is this needed?
```

Comment:
Does it not work without (it "ought to" after the `VUE_APP_SITE_ALIAS` pass...?)